pages/about.tsx

```diff
@@ -95,57 +81,6 @@ export default function About({ showNotification, loginContext }: AboutProps) {
           buttonLink={!loaded ? '/signup' : ''}
           disableButton={true}
         />
-        <div className={clsx('mx-6', 'px-3', 'w-full', 'lg:w-2/3', 'mb-6')}>
```

Comment:
Do we want to delete all these instructions? I feel like this is most of the FAQ for this page

Reply:
This is Phase 3 testnet guideline. Its on the phase3.testnet.ironfish.network.
I dont think we need the phase 3 tesnet info on the testnet website.